lib/models/database/database.g.dart

```diff
@@ -6290,7 +6291,8 @@ final class $$PlaylistMediaTableTableReferences extends BaseReferences<
       db.playlistTable.createAlias($_aliasNameGenerator(
           db.playlistMediaTable.playlistId, db.playlistTable.id));
 
-  $$PlaylistTableTableProcessedTableManager get playlistId {
+  $$PlaylistTableTableProcessedTableManager? get playlistId {
+    if ($_item.playlistId == null) return null;
```

Comment:
I'm guessing this was caused by `build_runner` but still please remove the changes in the file as it's unrelated to this PR